package/gluon-config-mode-geo-location-osm/check_site.lua

```diff
@@ -0,0 +1,4 @@
+need_number(in_site({'config_mode', 'geo_location', 'osm', 'center', 'lon'}))
+need_number(in_site({'config_mode', 'geo_location', 'osm', 'center', 'lat'}))
```

Comment:
Maybe we can have default values like 0.0/0.0 to reduce site.conf parameters.

Reply:
Certainly not. This is a value that *must* be provided by each site, 0.0/0.0 simply doesn't make sense.